core/deps/imgui/backends/imgui_impl_vulkan.cpp

```diff
@@ -502,11 +502,6 @@ void ImGui_ImplVulkan_RenderDrawData(ImDrawData* draw_data, VkCommandBuffer comm
         // Create or resize the vertex/index buffers
         size_t vertex_size = AlignBufferSize(draw_data->TotalVtxCount * sizeof(ImDrawVert), bd->BufferMemoryAlignment);
         size_t index_size = AlignBufferSize(draw_data->TotalIdxCount * sizeof(ImDrawIdx), bd->BufferMemoryAlignment);
-        if (vertex_size != 0)
-        	vertex_size = ((vertex_size - 1) / bd->BufferMemoryAlignment + 1) * bd->BufferMemoryAlignment;
-        if (index_size != 0)
-        	index_size = ((index_size - 1) / bd->BufferMemoryAlignment + 1) * bd->BufferMemoryAlignment;
-
```

Comment:
I guess this code was introduced to align the vertex and index sizes, and upstream fixed the issue later in lines 503 and 504 by using `AlignBufferSize`. So I believe this code can be safely removed but it would be nice if you can check/confirm it @flyinghead

Reply:
Yes, this isn't needed anymore